daily_read/order_portal.py

```diff
@@ -76,20 +89,53 @@ def process_orders(self, closed_before_in_days=30):
             if order["identifier"] in self.projects_data.data.keys():
                 proj_info = self.projects_data.data[order["identifier"]]
                 if order["reports"]:
-                    proj_prog_rep = next(item for item in order["reports"] if item["name"] == "Project Progress")
-                    proj_info.report_iuid = proj_prog_rep["iuid"]
-                if proj_info.orderer in order_updates:
-                    order_updates[proj_info.orderer]["projects"].append(proj_info)
-                else:
+                    prog_reports = [item for item in order["reports"] if item["name"] == "Project Progress"]
+                    if len(prog_reports) > 1:
+                        log.error(f"Multiple reports for Project Progress for order {order['identifier']}")
```

Comment:
What do we want to do in this case? It should be rare, but I think we should either handle this case or throw an exception? Just logging the error and moving on can have weird crashes further down in the code

Reply:
It'll raise an exception now